src/D2L.Bmx/WriteHandler.cs

```diff
@@ -70,6 +70,11 @@ bool useCredentialProcess
 			if( !data.Sections.ContainsSection( sectionName ) ) {
 				data.Sections.AddSection( sectionName );
 			}
+			var defaultCredentialsFile = parser.ReadFile( SharedCredentialsFile.DefaultFilePath );
+			if (defaultCredentialsFile.Sections.ContainsSection( profile ) ) {
+				defaultCredentialsFile.Sections.RemoveSection( profile );
+				parser.WriteFile(SharedCredentialsFile.DefaultFilePath, defaultCredentialsFile);
```

Comment:
Formatting looks off here.
Please check the auto formatting setup in your IDE and make sure it respects editorconfig. Let us know if you need any help with that.